### PR TITLE
Add MuxOH to Utils documentation

### PR DESCRIPTION
**Pull request by piessteady**

### CODE REVIEW

File: source/SpinalHDL/Libraries/utils.rst [hunk 1]
```diff
@@ -54,6 +54,24 @@ State less utilities
...
+       | MuxOH(
+       |   oneHot : BitVector,
+       |   inputs : Vec[T]
+       | )
+       | MuxOH(
+       |   oneHot : collection.IndexedSeq[Bool],
+       |   inputs : Vec[T]
+       | )
+     - T
+     - Returns the muxed ``T`` from the ``inputs`` based on the ``oneHot`` vector.
```
Comment (wifasoi) [inline source/SpinalHDL/Libraries/utils.rst]: Having to much line text in a table is a readability issue IMHO. I suggest to boild down the command example to just:
```
   * - | MuxOH(
       |   oneHot : IndexedSeq[Bool],
       |   inputs : Seq[T]
       | )
```

Comment (piessteady) [reply source/SpinalHDL/Libraries/utils.rst]: Ok. Is there by the way to link the documentation to the code?

Comment (wifasoi) [reply source/SpinalHDL/Libraries/utils.rst]: yes there it is.
There should be a scaladoc api documentation hosted in the main repo (here: http://spinalhdl.github.io/SpinalHDL).
So you can do this:
```
`MuxOH <http://spinalhdl.github.io/SpinalHDL/#spinal.lib.MuxOH$>`_
```
So MuxOH will be an hiperlink to the scaladoc reference.

**Comment by wifasoi** [comment]
Can i merge it, or you want to add the link tot he scaladoc page?

**Comment by piessteady** [comment]
I've added the link.

**Comment by piessteady** [comment]
@wifasoi the `linkcheck` fails because it is looking for the anchor `#spinal.lib.MuxOH$`
Do you know how to fix this?

**Comment by wifasoi** [comment]
I relaxed the linkcheck.
Thank for your contribution (and sorry for the linkcheck issue).

**Pull request merged**